rules/Renaming/Configuration/MethodCallRenameCollector.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Renaming\Configuration;
+
+use Rector\Renaming\Contract\MethodCallRenameInterface;
+
+final class MethodCallRenameCollector
```

Comment:
I think `*Collector` should be in a "Collector" namespace instead of `Configuration`

Reply:
OK I agree, this was inspired by [RenamedClassesDataCollector](https://github.com/rectorphp/rector/blob/28e1e1f3f4b0796b34105d1e2e26a3d66e74682a/src/Configuration/RenamedClassesDataCollector.php)

Reply:
done